controller/pages/api/util/pewpew.ts

```diff
@@ -74,6 +77,16 @@ export async function getPewpew (): Promise<ErrorResponse | PewPewVersionsRespon
   }
 }
 
+// Store these as maps for fast lookup oldest can be found by lastRequested, lastUpdated, and lastChecked
+// We can't use static variables since the memory spaces aren't shared between api and getServerSideProps
+// https://stackoverflow.com/questions/70260701/how-to-share-data-between-api-route-and-getserversideprops
+declare global {
+  // https://stackoverflow.com/questions/68481686/type-typeof-globalthis-has-no-index-signature
+  // eslint-disable-next-line no-var
```

Comment:
Looks like you do need to flip the order of these two lines. The `// eslint ...` needs to be right above the `var` line.